cas-server-integration-hazelcast/src/main/java/org/jasig/cas/ticket/registry/HazelcastTicketRegistry.java

```diff
@@ -151,8 +151,32 @@ private void deleteChildren(final TicketGrantingTicket ticket) {
             }
         }
     }
+    
+    @Override
+    public int sessionCount() {
+        int count = 0;
+        final Collection<Ticket> collection = getTickets();
```

Comment:
I'd love to, except that this is targeted for 4.2 and that branch is based on JDK7. 

Master will be stream-based though.